assets/css/about.css

```diff
@@ -0,0 +1,478 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap');
+
+@font-face {
+  font-family: 'Cocogoose';
+  src:
+    url('cocogoose_pro-trial-webfont.woff2') format('woff2'),
+    url('cocogoose_pro-trial-webfont.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+}
+
+#top-bar {
+  padding: 15px 50px 15px 230px;
+}
+
+header {
+  padding: 10px 50px 10px 230px;
+}
+
+#about-link {
+  color: #ec5242;
+}
+
+#headline {
+  background: url('../img/headline-bg.jpg') no-repeat right;
+  background-size: cover;
+  width: 100%;
+  height: 87vh;
+  padding: 20px 50px 20px 230px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  border-bottom: 1px solid #d3d3d3;
+}
+
+.heading h1 {
+  color: #ec5242;
+  font-family: 'Cocogoose', sans-serif;
+  font-weight: 800;
+  font-size: 75px;
+  line-height: 75px;
+  text-align: center;
+}
+
+.headline-info {
+  background: #fff;
+  padding: 20px 80px;
+}
+
+.headline-info p {
+  color: #272a31;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 20px;
+  text-align: center;
+}
+
+.headline-contact {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.headline-contact p {
+  color: #272a31;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 20px;
+  margin-bottom: 10px;
+  text-align: center;
+}
+
+.headline-contact a {
+  color: #272a31;
+  font-family: 'Cocogoose', sans-serif;
+  font-weight: 600;
+  font-size: 17px;
+  line-height: 20px;
+  text-decoration: underline;
+}
+
+@media screen and (max-width: 768px) {
+  #top-bar {
+    display: none;
+  }
+
+  header {
+    padding: 5px 30px;
+  }
+
+  #headline {
+    padding: 20px 20px;
+    justify-content: space-around;
+    align-items: center;
+  }
+
+  .heading h1 {
+    font-weight: 700;
+    font-size: 35px;
+    line-height: 40px;
+  }
+
+  .headline-info {
+    padding: 10px 20px;
+  }
+}
+
+#logo {
+  width: 100%;
+  padding: 30px 100px 30px 300px;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  border-bottom: 1px solid #d3d3d3;
+}
+
+.logo-heading h2 {
+  color: #272a31;
+  font-family: 'Cocogoose', sans-serif;
+  font-weight: 500;
+  font-size: 24px;
+  line-height: 30px;
+  letter-spacing: 0;
+}
+
+.logo-heading h2::after {
+  content: '';
+  display: block;
+  position: absolute;
+  width: 50px;
+  border-bottom: 3px solid #ec5242;
+  margin-left: 165px;
+  margin-top: 10px;
+}
+
+.logo-info {
+  margin-top: 30px;
+  margin-bottom: 30px;
+}
+
+.logo-info p {
+  color: #272a31;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 20px;
+  text-align: center;
+}
+
+.logo-img {
+  border: 1px solid #d3d3d3;
+  padding: 15px 40px;
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
- [ ] Kindly, please redesign your app using a mobile-first approach, as learned in the whole module that we finish now.